### PR TITLE
Create Planet model

### PR DESCRIPTION
**Pull request by LaureneEpp**

### CODE REVIEW

File: app/models/planet.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Planet < ApplicationRecord
+  belongs_to :user
+
+  validates :name, presence: true, uniqueness: true
+  validates :price, presence: true
```
Comment (timdavhen) [inline app/models/planet.rb:5]: I think the address and the description should be mandatory as well.

File: db/migrate/20210109124000_create_planets.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreatePlanets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :planets do |t|
+      t.string :name
+      t.string :description
```
Comment (timdavhen) [inline db/migrate/20210109124000_create_planets.rb]: Maybe change the description as text and not string.

**Review by timdavhen — CHANGES_REQUESTED** [review]
Good job, but few comments :)

File: db/migrate/20210109124000_create_planets.rb (hunk 2 above)
Comment (AlexandreDidion) [reply db/migrate/20210109124000_create_planets.rb]: Yup, shouldn't be a string

**Review by AlexandreDidion — CHANGES_REQUESTED** [review]
Like Tim said, otherwise nice!

**Pull request merged**